engine/src/main/java/org/terasology/logic/afk/AfkClientSystem.java

```diff
@@ -77,14 +79,11 @@ public void onCommand() {
         component.afk = !component.afk;
         entity.addOrSaveComponent(component);
         if (component.afk) {
-            nuiManager.pushScreen(SCREEN_URL, AfkScreen.class).setAfkClientSystem(this);
-            nuiManager.closeScreen(CONSOLE_SCREEN_URL);
             enableDiscord();
-            console.addMessage("[AFK] You are AFK now!");
+            logger.info("[AFK] You are AFK now!");
```

Comment:
I think some form of confirmation would be nice... Console is the text box in the bottom left, right? I think we could also print to the dev console by returning a String from this command method.